components/pokemonapp.brs

```diff
@@ -36,7 +37,22 @@ function onPokemonFocus(event as Object)
     updateText(selectedPokemon)
 end function
 
+function onPokemonSelect(event as Object)
+    data = event.GetData()
+    pokemonIndex = data[1]
+    selectedPokemon = m.pokeList.content.getChild(0).getChild(pokemonIndex)
+    pokemonSplashArt = CreateObject("roSGNode", "PokemonSplashArt")
+    createPokemonSplashArt(pokemonSplashArt, selectedPokemon)
+end function
+
 function updateText(selectedPokemon as Object)
     m.titleLabel.text = selectedPokemon.title
     m.descriptionLabel.text = selectedPokemon.pokemonDescription
 end function
+
+function createPokemonSplashArt(splashArtObject as Object, selectedPokemon as dynamic)
+    splashArtObject.contenturi = selectedPokemon.pokemonBGImage
+    m.pokeList.setFocus(false)
```

Comment:
In line 57 you set focus true on `splashArtObject`. This line is not needed because the focus is automatically lost from pokeList